docs/csharp/how-to/safely-cast-using-pattern-matching-is-and-as-operators.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "How to: safely cast by using pattern matching is and as operators"
```

Comment:
I think one more "and" would make the title easier to read:
"How to: safely cast by using pattern matching and is and as operators"